src/fabric.c

```diff
@@ -469,6 +495,28 @@ static int lib_filter(const struct dirent *entry)
 }
 #endif
 
+static void verify_filter_names(char **names)
+{
+	int i, j;
+	char** split_names;
+	for (i = 0; names[i]; i++) {
+		split_names = ofi_split_and_alloc(names[i], ";", NULL);
+		if (!split_names)
+			FI_WARN(&core_prov, FI_LOG_CORE,
+				"unable to parse given filter string\n");
```

Comment:
Need to return here, or we'll crash trying to dereference split_names.

Reply:
@shefty done